include/lsst/meas/modelfit/UnitSystem.h

```diff
@@ -48,9 +49,9 @@ struct UnitSystem {
      *  set such that unit flux is the given magnitude.  See @ref modelfitUnits for an explanation
      *  of why we frequently use this system.
      */
-    UnitSystem(afw::coord::IcrsCoord const & position, std::shared_ptr<const lsst::afw::image::Calib> calibIn,
+    UnitSystem(afw::geom::SpherePoint const& position, std::shared_ptr<const lsst::afw::image::Calib> calibIn,
```

Comment:
No need to move the ```&```.  Same below.

Reply:
clang-format; I can make it a separate commit if you want

Reply:
My concern was for consistency in the file/package.  Did it flag the other cases in this file?

Reply:
I can run clang-format on the file if you prefer. Normally I only run it on code I change.

Reply:
I prefer to keep the same style within a file, but I'm happy to be overruled if this is not what others want.

Reply:
Fair enough. I'll run clang-format as a separate commit.